### PR TITLE
cleanup(core): refactored @nrwl/workspace command-line to improve code quality

### PR DESCRIPTION
**Pull request by Phillip9587**
## Related Issue(s)

Fixes #

### CODE REVIEW

File: packages/workspace/src/command-line/workspace-generators.ts [hunk 1]
```diff
@@ -464,10 +447,8 @@ function createTmpTsConfig(
   return tmpTsConfigPath;
 }
 
-function cleanupTmpTsConfigFile(tmpTsConfigPath) {
-  try {
```
Comment (FrozenPandaz) [inline packages/workspace/src/command-line/workspace-generators.ts:468]: I think this `try catch` is intentional. Failures here are okay. The `tmp` file will remain on the system but will be overwritten the next time it is needed.

File: packages/workspace/src/command-line/dep-graph.ts [hunk 2]
```diff
@@ -161,7 +163,7 @@ export function generateGraph(
     return a.name.localeCompare(b.name);
   });
 
-  if (args.focus !== undefined) {
+  if (args.focus) {
```
Comment (FrozenPandaz) [inline packages/workspace/src/command-line/dep-graph.ts:166]: @philipjfulcher can you check if these `!== undefined` are needed here?

**Review by FrozenPandaz — CHANGES_REQUESTED** [review]
Thank you! Looks much cleaner :smile:.

There are some minor things that shouldn't have changed.

File: packages/workspace/src/command-line/dep-graph.ts (hunk 2 above)
Comment (philipjfulcher) [reply packages/workspace/src/command-line/dep-graph.ts:166]: Confirmed they're not needed. :+1:

**Comment by FrozenPandaz** [comment]
Please rebase and fix the merge conflicts. :pray:

**Pull request merged**